lib/ansible/modules/windows/win_service.ps1

```diff
@@ -409,6 +576,10 @@ Function Set-ServiceConfiguration($svc) {
     if ($null -ne $state) {
         Set-ServiceState -svc $svc -wmi_svc $wmi_svc -state $state
     }
+
+    if (($null -ne $recovery_actions) -and ($null -ne $recovery_reset_interval)) {
+        Set-ServiceRecovery -name $svc.Name -reset_interval $recovery_reset_interval -actions $recovery_actions
```

Comment:
Why not allow setting either one?

Reply:
Well the thing is, if you set just one with sc.exe, it sets the same actions for the two other actions too.
If you set two, the last actions has the same value as the second one.
Unless I tested improperly. The behaviour is different when you set it in the UI.

We could allow for this to work if we used the registry to set the values, but @jborean93 was strongly disagreeing with that.